guides/v2.3/graphql/develop/create-graphqls-file.md

```diff
@@ -6,7 +6,7 @@ github_link: graphql/develop/create-graphqls-file.md
 redirect_from: graphql/develop/configure-graphql-xml.html
 ---
 
-Each module that adds to or extends from a GraphQL schema can do so by placing a `schema.graphqls` file in its `etc` directory. Magento Core adds GraphQl modules based on the purpose of the schema being extended/added and the core modules they depend on. For example, the CustomerGraphQl module adds a query to the graphql endpoint to view customer data and relies on the Customer core module.
+Each module that adds to or extends from a GraphQL schema can do so by placing a `schema.graphqls` file in its `etc` directory. Magento Core adds [`GraphQl`](https://github.com/magento/magento2/tree/2.3-develop/app/code/Magento/GraphQl) modules based on the purpose of the schema being extended/added and the core modules they depend on. For example, the CustomerGraphQl module adds a query to the graphql endpoint to view customer data and relies on the Customer core module.
```

Comment:
Again, change the URLs to https://github.com/magento/magento2/blob/{{ page.guide_version }}-develop

The link in line 9 is invalid.  However, if you want to add a link to CustomerGraphQl, that would be valid.